src/Suave/Owin.fs

```diff
@@ -634,8 +634,17 @@ module OwinApp =
         HttpContext.request_ >--> HttpRequest.headers_,
         HttpContext.response_ >--> HttpResult.headers_
 
+      let handleResponse  (rhs : Dictionary<string, string[]>) = 
+        let handleKey key =
```

Comment:
Sure, that's a good idea to save a lot of allocations... In general hoisting such strings to statics to save allocations is probably a good idea.